frontend/src/components/worksheets/items/ActionButtons.js

```diff
@@ -182,6 +185,20 @@ class ActionButtons extends React.Component<{
                         Schema
                     </Button>
                 </Tooltip>
+                <Tooltip title='Add an image'>
+                    <Button
+                        size='small'
+                        color='inherit'
+                        aria-label='image'
+                        onClick={uploadNewImage}
```

Comment:
I don't see the point in passing `uploadNewImage` as a prop from a parent many levels above to this component, given that its implementation just calls a js function that isn't dependent on state (`document.querySelector('label[for=codalab-image-upload-input]').click();
`). Could you just define `uploadNewImage` within this component instead? That would simplify it.

Reply:
thanks for the suggestion! Updated.